THIRD-PARTY-NOTICES.md

```diff
@@ -7600,27 +7600,48 @@ License tags: MIT
 License files:
```

Comment:
This should not be changed manually, seems like something might've gone wrong during the merge/rebase?

Reply:
Yap, seems like that is what happened.